cmd/manager/aggregator.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Was this just a formatting thing that ended up in this PR? Or was this necessary for this change?

Reply:
Yep, this one was by accident. Sorry about that!

Reply:
I think this was from `make generate`